Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileTransporter.cs

```diff
@@ -444,10 +448,22 @@ private SingleFileTransferResult NoSourceOperation(BatchContext context, Source
             case SourceAction.Ignore:
                 return new SingleFileTransferResult { Success = true, ActionSkipped = true, ErrorMessages = { msg } };
             default:
-                throw new Exception("Unknown operation in NoSourceOperation");
+                throw new Exception("Unknown operation in NoSourceOperation.");
         }
     }
 
+    /// <summary>
+    /// Handles logging of actions.
+    /// </summary>
+    /// <param name="state"></param>
+    /// <param name="format"></param>
+    /// <param name="args"></param>
+    private void Trace(TransferState state, string format, params object[] args)
+    {
+        State = state;
+        _logger.NotifyTrace(string.Format("{0}: {1}", state, string.Format(format, args)));
```

Comment:
Also, can we drop the string.Format altogether? :) Cobalt was written when this was a viable way to interpolate strings, but now you can just use $"Hello, {var}", instead of the string.Format("Hello, {0}", var); 

Maybe no need to go for the entire code base in this PR, but at least let's not introduce new string.Formats into code?